tests/integration_test.go

```diff
@@ -221,7 +221,8 @@ func (s *IntegrationTestSuite) testSuccessOneSwap(
 	awsc := s.newSwapdWSClient(ctx, defaultXMRTakerSwapdWSEndpoint)
 
 	// TODO: implement discovery over websockets (#97)
-	peerIDs, err := ac.Discover(coins.ProvidesXMR, defaultDiscoverTimeout)
+	time.Sleep(time.Second * 3)
```

Comment:
Can you add a short comment stating what we are waiting for?

Reply:
How about?
```
// Give the maker's p2p offer advertisement time to propagate
time.Sleep(time.Second * 3)
```
I'm going to try to describe the full problem, that doesn't need to be in the comment, and you can tell me where I'm wrong:

When the maker creates an offer, he then advertises in the libp2p distributed hash table that he has one or more offers. This can take a few seconds to propagate even when all the peers are on the same localhost.

Reply:
comment updated

Reply:
Was my description above correct?